nipype/interfaces/mrtrix3/utils.py

```diff
@@ -108,7 +108,49 @@ def _list_outputs(self):
         return outputs
 
 
-class Generate5ttInputSpec(CommandLineInputSpec):
+class Generate5ttInputSpec(MRTrix3BaseInputSpec):
```

Comment:
This interface is related to the command 5ttgen that allows the use of threads and the related option, so I think it makes sense to use MRTrix3BasedInputSpec as base class rather than CommandLineInputSpec.